### DISCUSSION TITLE
Add Independent (and Manual) Sort Order for Each Tab

### DISCUSSION THREAD

**Discussion by brucetheelephant** [mateusz-bak/openreads#702] [Ideas]
**Is your feature request related to a problem? Please describe.**
When you choose a Sort Order, that order is applied all tabs.

If I choose to sort by Start Date in the In Progress tab then it will be applied to the In Progress tab and to the Finished tab as well. If I want to sort by Finish Date in the Finished tab then I will have to resort books. And then if I move back to the In Progress tab where I want the books to be sorted by Start Date then I will have to resort yet again.

In the For Later tab I'm not even sure how books are being sorted.

This is very frustrating.

**Describe the solution you'd like**
I would like to be able to change the Sort Order for each tab individually and I would like to be able to sort books manually in the For Later tab.

**Comment by armand-leguillou** [comment]
i miss that too!

**Comment by mateusz-bak** [comment]
Implemented in https://github.com/mateusz-bak/openreads/pull/823 by @k12onoss

**Reply by brucetheelephant** [reply]
Thank you @k12onoss and @mateusz-bak!
